src/common/card/Card.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import styles from './card.module.scss';
+
+const Card = ({
+  width,
+  Header,
+  children,
+}: {
+  width: string;
+  Header: React.FC;
+  children: React.ReactChildren;
+}) => (
+  <div style={{ width }}>
```

Comment:
As we discussed, can we try to avoid inline styles as possible, I'd suggest removing this `width`, or better yet, the whole outer wrapper.

Reply:
I removed the outer wrapper.